app/views/templates/rulingCoverLetterTemplate.scala.html

```diff
@@ -136,21 +136,90 @@ <h3 class="govuk-heading-m">@messages("pdf.ruling.section.ifYouDisagree.title")<
+                @Html(messages("pdf.ruling.section.askForReview.solsTeam.emailText", messages("pdf.ruling.section.askForReview.solsTeam.email"), messages("pdf.ruling.section.askForReview.solsTeam.email")))
+            </p>
+            <p class="govuk-body-s">
+                @messages("pdf.application.footer.solicitors.office.and.legal.services.address")
+            </p>
+            <h4 class="govuk-heading-s">@messages("pdf.ruling.section.appealForIndependentTribunal.title")</h4>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.appealForIndependentTribunal.appealImmediately")
+            </p>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.appealForIndependentTribunal.courtsAndTribunals")
+            </p>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.appealForIndependentTribunal.requestRequirements")
+            </p>
+            <ul class="govuk-list govuk-list--bullet govuk-body-s">
+                <li>@messages("pdf.ruling.section.appealForIndependentTribunal.requestRequirements.1")</li>
+                <li>@messages("pdf.ruling.section.appealForIndependentTribunal.requestRequirements.2")</li>
+            </ul>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.appealForIndependentTribunal.relevantInfo")
+            </p>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.appealForIndependentTribunal.tribunalsInfo")
+            </p>
+            <p class="govuk-body-s">
+                @Html(messages("pdf.ruling.section.appealForIndependentTribunal.page"))
+            </p>
+            <h4 class="govuk-heading-s">@messages("pdf.ruling.section.appealsAndReviews.title")</h4>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.appealsAndReviews.moreInfo")
+            </p>
+            <p class="govuk-body-s">
+                @Html(messages("pdf.ruling.section.appealForIndependentTribunal.page"))
+            </p>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.appealsAndReviews.copy")
+            </p>
+            <h4 class="heading-small">@messages("pdf.ruling.section.communicatingByEmail.title")</h4>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.communicatingByEmail.tellUs")
+            </p>
+            <p class="govuk-body-s">
+                @Html(messages("pdf.ruling.section.communicatingByEmail.page"))
             </p>
             <p class="govuk-body-s">
-                @messages("pdf.ruling.section.askForReview.acceptedReview")
+                @messages("pdf.ruling.section.communicatingByEmail.copy")
             </p>
-            <h4 class="govuk-heading-s">@messages("pdf.ruling.section.appealing.title")</h4>
+            <h4 class="heading-small">@messages("pdf.ruling.section.acknowledgement.title")</h4>
```

Comment:
the class `heading-small` is formatting it weirdly and displaying it differently to other h4s. Please replace with `govuk-heading-s` instead like the other h4s, thanks